components/automate-ui/src/app/components/page-picker/page-picker.component.html

```diff
@@ -17,7 +17,7 @@
       class="page-picker-item"
       [attr.primary]="item === page"
       [attr.secondary]="item !== page"
-      [attr.disabled]="selectablePages.length <= 1"
+      [attr.disabled]="selectablePages.length <= 1 && last <= 1"
```

Comment:
Similar to the addition above, this makes sure that the page selector button is _not_ disabled when we hit a scenario where there is only one selectable page showing, but may still have multiple pages.

Reply:
Nice!